caddyfile/dispenser_test.go

```diff
@@ -198,6 +198,47 @@ func TestDispenser_NextBlock_MakesProgressOrStopsOnEOF(t *testing.T) {
 	t.Fatalf("NextBlock(): exceeded %d iterations without terminating", maxIters)
 }
 
+func TestDispenser_NextBlock_MultipleLevelsOfNestedBlocks(t *testing.T) {
+	input := `foobar1 {
+			    sub1 arg1
+			    sub2
+			    sub_foobar1 sub_foobar1_arg {
+			      sub3 arg3
+			      sub4
+			    }
```

Comment:
I'm still seeing mixed whitespace here:
<img width="563" height="199" alt="Screenshot 2025-12-27 at 15 03 42" src="https://github.com/user-attachments/assets/0cd6d57b-4455-48fc-8fdc-7f6994e09e91" />


Here's a suggestion that should get it right:

```suggestion

	input := `foobar1 {
                sub1 arg1
                sub2
                sub_foobar1 sub_foobar1_arg {
                  sub3 arg3
                  sub4
                }
```

Reply:
Other tests such as TestDispenser_Args and TestDispenser_NextLine also use a mixed tab/space format, so I kept it consistent with the current codebase.

 But I can change it if you prefer, please confirm.

Reply:
Ah, alright. This will do just fine then!